src/Character/Character.jsx

```diff
@@ -43,10 +86,43 @@ export default function Character() {
     return attributeList;
   };
 
+  const renderClassOptions = () => {
+    const classList = [];
+    const classNames = Object.keys(classes);
+    for (let cName of classNames) {
```

Comment:
Nit: This is maybe my personal style preference, but I would do a `map` and spell out `className`.

`const classList = classNames.map((className) => <option key={sha256(className)}>{capitalize(className)}</option>);`